snuba/query/conditions.py

```diff
@@ -86,7 +86,15 @@ def __set_condition_pattern(
         String(operator),
         (
             Param("lhs", lhs),
-            Param("tuple", FunctionCallPattern(String("tuple"), None)),
+            Param(
+                "tuple",
```

Comment:
the name of this param is a little misleading. maybe let's call it `collection` or `sequence`?